barretenberg/cpp/src/barretenberg/relations/elliptic_relation.hpp

```diff
@@ -67,31 +67,36 @@ template <typename FF_> class EllipticRelationImpl {
         auto y1_sqr = (y_1 * y_1);
         auto y1y2 = y_1 * y_2 * q_sign;
         auto x_add_identity = (x_3 + x_2 + x_1) * x_diff * x_diff - y2_sqr - y1_sqr + y1y2 + y1y2;
-        std::get<0>(accumulators) += x_add_identity * scaling_factor * q_elliptic * (-q_is_double + 1);
+
+        auto q_elliptic_by_scaling = q_elliptic * scaling_factor;
+        auto q_elliptic_q_double_scaling = q_elliptic_by_scaling * q_is_double;
```

Comment:
is (1-q_is_double) the same as q_is_double here always?

Reply:
No. I compute q_elliptic_not_double_scaling separately at line 73